client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/faqs.tsx

```diff
@@ -138,6 +140,32 @@ const DomainTransferFAQ: FC = () => {
 							) }
 						</FoldableFAQ>
 					</li>
+					<li>
+						<FoldableFAQ
+							id="privacy-protection"
+							question={ translate(
+								'It looks like I am being charged, I thought the first year was free?'
+							) }
+							onToggle={ onFaqToggle }
+							className="domain-transfer-faq__section"
+						>
+							{ translate(
+								'We offer a free first year for all domains being transferred in, but some domain extensions may not ' +
```

Comment:
This offer only applies to domains that are coming from Google Domains.

Reply:
👍